src/DssSpell.sol

```diff
@@ -36,40 +55,107 @@ contract DssSpellAction is DssAction {
+    address constant RWA002_GEM              = 0xAAA760c2027817169D7C8DB0DC61A2fb4c19AC23;
+    address constant MCD_JOIN_RWA002_A       = 0xe72C7e90bc26c11d45dBeE736F0acf57fC5B7152;
+    address constant RWA002_A_URN            = 0x225B3da5BE762Ee52B182157E67BeA0b31968163;
+    address constant RWA002_A_INPUT_CONDUIT  = 0x2474F297214E5d96Ba4C81986A9F0e5C260f445D;
+    address constant RWA002_A_OUTPUT_CONDUIT = 0x2474F297214E5d96Ba4C81986A9F0e5C260f445D;
 
     function actions() public override {
-        // Increase the COMP-A Maximum Debt Ceiling from 10M to 30M
-        DssExecLib.setIlkAutoLineDebtCeiling("COMP-A", 30 * MILLION);
-
-        // Increase the ZRX-A Maximum Debt Ceiling from 5M to 10M
-        DssExecLib.setIlkAutoLineDebtCeiling("ZRX-A", 10 * MILLION);
-
-        // Increase the YFI-A Maximum Debt Ceiling from 45M to 75M
-        DssExecLib.setIlkAutoLineDebtCeiling("YFI-A", 75 * MILLION);
-
-        // Increase the PSM-USDC-A Debt Ceiling from 1B to 2B
-        DssExecLib.increaseIlkDebtCeiling("PSM-USDC-A", 1 * BILLION, true);
-
-        // Provide Core Unit Budgets
-
-        // Real-World Finance + Governance (Interim Multi-Sig)
-        DssExecLib.sendPaymentFromSurplusBuffer(INTERIM_MULTISIG, (40 + 80) * THOUSAND);
-
-        // Risk
-        DssExecLib.sendPaymentFromSurplusBuffer(RISK, 100_500);
+        bytes32 ilk   = "RWA002-A";
+        uint256 CEIL  = 5 * MILLION * WAD;
+        uint256 PRICE = 5_634_804 * WAD;
+        uint256 MAT   = 10_500;
+        uint48 TAU    = 0;
+
+        // https://ipfs.io/ipfs/QmdfuQSLmNFHoxvMjXvv8qbJ2NWprrsvp5L3rGr3JHw18E
+        string memory DOC = "QmdfuQSLmNFHoxvMjXvv8qbJ2NWprrsvp5L3rGr3JHw18E";
+
+        address MIP21_LIQUIDATION_ORACLE =
+            DssExecLib.getChangelogAddress("MIP21_LIQUIDATION_ORACLE");
+
+        address vat = DssExecLib.vat();
+
+        // Sanity checks
+        require(GemJoinAbstract(MCD_JOIN_RWA002_A).vat() == vat, "join-vat-not-match");
+        require(GemJoinAbstract(MCD_JOIN_RWA002_A).ilk() == ilk, "join-ilk-not-match");
+        require(GemJoinAbstract(MCD_JOIN_RWA002_A).gem() == RWA002_GEM, "join-gem-not-match");
+        require(GemJoinAbstract(MCD_JOIN_RWA002_A).dec() == DSTokenAbstract(RWA002_GEM).decimals(), "join-dec-not-match");
+
+        RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).init(
+            ilk, PRICE, DOC, TAU
+        );
+        (,address pip,,) = RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).ilks(ilk);
+
+        // Set price feed for RWA002
+        DssExecLib.setContract(DssExecLib.spotter(), ilk, "pip", pip);
+
+        // Init RWA-002 in Vat
+        Initializable(vat).init(ilk);
+        // Init RWA-002 in Jug
+        Initializable(DssExecLib.jug()).init(ilk);
+
+        // Allow RWA-002 Join to modify Vat registry
+        DssExecLib.authorize(vat, MCD_JOIN_RWA002_A);
+
+        // Allow RwaLiquidationOracle to modify Vat registry
+        // DssExecLib.authorize(vat, MIP21_LIQUIDATION_ORACLE);
```

Comment:
Uncomment as the Oracle needs to `grab` bad debt

Reply:
it was already done in the MIP21 spell.